### ISSUE TITLE
Cannot find module 'AccessibilityInfo'

### ISSUE THREAD

**Issue by svvashishtha** [FaridSafi/react-native-google-places-autocomplete#213]
I installed this module using 
`npm install react-native-google-places-autocomplete --save`

as mentioned in [docs].(https://github.com/FaridSafi/react-native-google-places-autocomplete#installation)

But on adding the import line 
`import { GooglePlacesAutocomplete } from 'react-native-google-places-autocomplete';`

I get the error:

> error: bundling failed: "TransformError: _Project_name_/index.android.js: Cannot find module 'AccessibilityInfo' (While processing preset: \"/_Project_name_/node_modules/react-native/Libraries/react-native/react-native-implementation.js\")"

**Comment by svvashishtha** [comment]
As this issue is very common. **It might not have anything to do with this particular module**. As mentioned here(https://github.com/facebook/react-native/issues/14209), there are multiple ways to solve this. 
I ended up uninstalling react-native-google-places-autocomplete using 
`npm uninstall react-native-google-places-autocomplete`

then followed the installation instructions after installing all peer dependencies and got my project to work.
